src/quic/node_quic_http3_application.h

```diff
@@ -133,7 +133,7 @@ class Http3Application final :
   void MemoryInfo(MemoryTracker* tracker) const override;
 
  private:
-  nghttp3_conn* Connection() { return connection_.get(); }
+  nghttp3_conn* Connection() const { return connection_.get(); }
```

Comment:
Not sure, but maybe this is a good PR to also align the getter naming with our style guide and use `connection()`, `session()`, etc.?

Reply:
yeah, but let me land the other opens ones first, then I'll rebase and make that change